h/renderers.py

```diff
@@ -0,0 +1,31 @@
+# Taken from https://pyramid-cookbook.readthedocs.org/en/latest/templates/customrenderers.html
+# with minor modifications
+import csv
+try:
+    import StringIO as io # python 2
+except ImportError:
+    from io import StringIO # python 3
+
+class CSVRenderer(object):
+   def __init__(self, info):
```

Comment:
Looks good. I think I'd be inclined to call this simply `CSV` given that it's in a module called `renderers`, as that leads to rather more fluent code:

```python
from h.renderers import CSV
```

or 

```python
from h import renderers

...

renderers.CSV
```